geode-core/src/main/java/org/apache/geode/management/internal/i18n/CliStrings.java

```diff
@@ -2557,14 +2557,14 @@
       "The gemfire.properties file for configuring the Cache Server's distributed system. The file's path can be absolute or relative to the gfsh working directory.";
   public static final String START_SERVER__REDIS_PORT = ConfigurationProperties.REDIS_PORT;
   public static final String START_SERVER__REDIS_PORT__HELP =
-      "Sets the port that the Geode Redis service listens on for Redis clients.";
+      "Sets the port that the Redis API for Geode server listens on for Redis clients. The default is an ephemeral port";
```

Comment:
For gfsh start server I thought the default would be 6379

Reply:
Yes!